examples/simple/package.json

```diff
@@ -5,7 +5,6 @@
     },
     "dependencies": {
         "@pulumi/aws": "^6.0.0",
-        "@pulumi/aws-apigateway": "1.1.0-alpha.3",
```

Comment:
No problem. I didn't think we needed it because of it being placed but I'll update it to what you suggested and do the same for the go and python examples too

Reply:
Actually python looks fine because it references the SDK folders anyway